homeassistant/components/remote/roku.py

```diff
@@ -0,0 +1,64 @@
+"""
+Support for the Roku remote.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/remote.roku/
+"""
+
+from homeassistant.components import remote
+from homeassistant.const import (CONF_HOST)
+
+
+DEPENDENCIES = ['roku']
+
+
+async def async_setup_platform(hass, config, async_add_entities,
+                               discovery_info=None):
+    """Set up the Apple TV remote platform."""
+    if not discovery_info:
+        return
+
+    host = discovery_info[CONF_HOST]
+    async_add_entities([RokuRemote(host)])
+
+
+class RokuRemote(remote.RemoteDevice):
+    """Device that sends commands to an Apple TV."""
```

Comment:
This should probably be `Roku` and not `Apple`

Reply:
Fixed :)